docs/pages/desktop-access/reference/sessions.mdx

```diff
@@ -87,4 +76,14 @@ player supports toggling between play and pause, but does not support seeking to
 a specific point in the stream, rewinding, or restarting playback when the end
 of the stream is reached. To replay a session, refresh the page.
 
+## Storage
```

Comment:
@zmb3 should we add the results of the size estimate somewhere here?

500 bytes per 64x64 png (gzipped on disk) 
which means 250 kb per full frame change on 1080p

Reply:
Sure! Might as well!